src/V8.rs

```diff
@@ -12,13 +12,18 @@ use crate::support::UniqueRef;
 extern "C" {
   fn v8__V8__SetFlagsFromCommandLine(argc: *mut c_int, argv: *mut *mut c_char);
   fn v8__V8__SetFlagsFromString(flags: *const u8, length: usize);
+  fn v8__V8__SetEntropySource(callback: extern "C" fn(*mut u8, usize) -> bool);
   fn v8__V8__GetVersion() -> *const c_char;
   fn v8__V8__InitializePlatform(platform: *mut Platform);
   fn v8__V8__Initialize();
   fn v8__V8__Dispose() -> bool;
   fn v8__V8__ShutdownPlatform();
 }
 
+/// EntropySource is used as a callback function when v8 needs a source
+/// of entropy.
+pub type EntropySource = fn(&mut [u8]) -> bool;
```

Comment:
The V8 docs don't mention it but returning `false` makes V8 fall back to system entropy (`/dev/urandom`, etc.)